### DISCUSSION TITLE
Is there an equivalent method of iOS' `CreateViewControllerFor()` in Android?

### DISCUSSION THREAD

**Discussion by erni-ph-mobile-team** [MvvmCross/MvvmCross#4173] [Q&A]
In iOS, I can create a view by passing a `MvxViewModelRequest` to `CreateViewControllerFor`.

Is there an equivalent convenient method on Android?

**Comment by Cheesebaron** [comment]
Not really. Android works a bit different. When you want to create an Activity, you go through an Intent. However for Fragments, there is `CreateFragment` in the Android Presenter.

What are you trying to achieve?
